docs/moz-addon-manager.md

```diff
@@ -53,24 +53,29 @@ It is possible to serve the local development version of this project with HTTPS
    - Choose the file you have downloaded before (`example.com.ca.crt.pem`)
    - Accept this CA certificate to trust websites
 
-3. Start this project with the command below:
+3. In your custom Firefox profile, go to `about:config`, void the warranty, and set these prefs. Afterwards, restart Firefox for them to take effect.
+
+    - set `extensions.webapi.testing` to `true` [to turn on `mozAddonManager`](#turning-on-mozaddonmanager-in--dev-and--stage-environments)
+    - set `xpinstall.signatures.dev-root` to `true` [to install add-ons](#install-add-ons-in--dev-and--stage-environments)
+
+4. Start `addons-frontend` with the command below:
```

Comment:
Well, actually it only works for AMO, not Disco Pane (yet).

Reply:
OK, I'll add it to the known issues.